sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -577,10 +577,10 @@ object SQLConf {
     .doc("The maximum lines of a single Java function generated by whole-stage codegen. " +
       "When the generated function exceeds this threshold, " +
       "the whole-stage codegen is deactivated for this subtree of the current query plan. " +
-      "The default value 2667 is the max length of byte code JIT supported " +
-      "for a single function(8000) divided by 3.")
+      "The default value 4000 is the max length of byte code JIT supported " +
+      "for a single function(8000) divided by 2.")
     .intConf
-    .createWithDefault(2667)
```

Comment:
YES.... Just double check it. The default of `HugeMethodLimit`is 8000... Thanks!